manywheel/deploy.sh

```diff
@@ -2,16 +2,28 @@
 
 set -eou pipefail
 
-build_image() {
-    BASE_CUDA_VERSION=$1
-}
+for rocm_version in 3.7; do
+    (
+        set -x
+        DOCKER_BUILDKIT=1 docker build \
+            -t "pytorch/manylinux-rocm${rocm_version}" \
```

Comment:
Can we change this so that it's just a `manylinux-rocm` image with separate tags instead of having separate images for each rocm version?
```suggestion
            -t "pytorch/manylinux-rocm:${rocm_version}" \
```